components/src/organisms/quantity-update/QuantityUpdate.js

```diff
@@ -0,0 +1,106 @@
+import AInputWrapper from '../../molecules/input-wrapper/InputWrapper.vue'
+import AIcon from '../../atoms/icon/Icon.vue'
+import AButton from '../../atoms/button/Button.vue'
+import ALabel from '../../atoms/label/Label.vue'
+
+// @vue/component
+export default {
+  components: {
+    AInputWrapper,
+    AIcon,
+    AButton,
+    ALabel
+  },
+  props: {
+    /**
+     * Quantity
```

Comment:
Maybe dafault value?

Reply:
Updated to `Quantity value (used with v-model)`